src/fosslight_reuse/_add.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 LG Electronics Inc.
+# SPDX-License-Identifier: GPL-3.0-only
+import os
+import re
+import logging
+import fosslight_util.constant as constant
+from yaml import safe_dump
+from fosslight_util.set_log import init_log
+from datetime import datetime
+from ._fosslight_reuse import reuse_for_project, reuse_for_files, print_error
+from reuse.header import run
+from reuse._comment import EXTENSION_COMMENT_STYLE_MAP_LOWERCASE
+from reuse._main import parser as reuse_arg_parser
+
+_PKG_NAME = "fosslight_reuse"
+_check_only_file_mode = False
+_auto_add_mode = False
+_result_log = {}
+logger = logging.getLogger(constant.LOGGER_NAME)
+
+
+def check_file_extension(file_list):
+    files_filtered = []
+
+    if file_list != "":
+        for file in file_list:
+            file_extension = os.path.splitext(file)[1]
```

Comment:
An exception is required when there is no file extension.
- ex. file name = "test"

Reply:
okay. I agree with you, so I'll add exception line. Thank you